WooCommerce/Classes/ViewRelated/Coupons/CouponDetails/CouponDetails.swift

```diff
@@ -256,11 +256,23 @@ struct CouponDetails: View {
                 Text(String.localizedStringWithFormat(Localization.maximumSpend, viewModel.maximumAmount))
                     .renderedIf(viewModel.maximumAmount.isNotEmpty)
 
+                Text(String.localizedStringWithFormat(Localization.singularUsageLimitPerCoupon, viewModel.usageLimit))
+                    .renderedIf(viewModel.usageLimit == 1)
+
+                Text(String.localizedStringWithFormat(Localization.pluralUsageLimitPerCoupon, viewModel.usageLimit))
+                    .renderedIf(viewModel.usageLimit > 1)
+
```

Comment:
I'm not sure if the limits info I added here should be within the rendering constraints of this VStack, but I decided to follow the info order used in the Android implementation as a reference, also considering that the other limit info that was already here is also inserted inside this VStack. Let me know if this doesn't make sense.

Reply:
The reason why I added the condition check to the outer VStack was that the VStack was defined with spacing, so if none of the details in this VStack is rendered, we still get a small space like so:

<img src="https://user-images.githubusercontent.com/5533851/167989704-cb3e144a-4a83-46e7-93fd-1b2b396dda71.png" width=320 />

So, to be on the safe side, we need the conditions on both the views and the containing VStacks 😅 

An alternative solution would be to add padding to each of the items and remove the spacing for the VStack, but we'll end up with some duplicated code too. There's no perfect solution here to be honest.